### ISSUE TITLE
Endless fetching data

### ISSUE THREAD

**Issue by zoomtronic1** [Jorgen-VikingGod/ESP8266-WiFi-Relay#1]
It fetches data continuously, , not synchronizing web page on PC and on phone, when i click on  button for relay, it works, but it does not show change state of button on phone browser and vice versa

**Comment by Jorgen-VikingGod** [comment]
You are right - did not have the requirement yet, to bidirectional changes.
Feel free to fork and add this feature. I think I have only in about 2 month time again to work on my hobby projects - really busy at this time :/ 
But thanks for reporting - will add some tags to this issue.

**Comment by amyth1884** [comment]
I understand, taking time out is so difficult, I am trying to achieve the same and its very frustrating, trying to integrate ir as well, no luck so far
